<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>746</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>746</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qalqulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(61, 56, 70);
}

QPushButton {
	min-width: 50px;
	min-height: 50px;
	max-width: 50px;
	max-height: 50px;
	background-color: rgb(94, 92, 100);
	border: 2px solid rgb(61, 56, 70);
	border-radius: 25px;
	font-weight: bold;
	color: rgb(255, 255, 255);
	font: 700 14pt &quot;Ubuntu Sans&quot;;
}

QPushButton:hover {
	background-color: rgb(119, 118, 123);
}

QPushButton[operator=&quot;true&quot;] {
	background-color: rgb(36, 31, 49);
}

QPushButton[operator=&quot;true&quot;]:hover {
	background-color: rgb(94, 92, 100);
}

QPushButton:pressed {
	background-color: rgb(153, 193, 241);
	border: 3px solid rgb(53, 132, 228);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>440</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>400</width>
      <height>522</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="numbersLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item row="2" column="4">
      <widget class="QPushButton" name="buttonMultiply">
       <property name="text">
        <string>x</string>
       </property>
       <property name="operator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="button3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="buttonDot">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="button0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="button8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="button2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="button6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="button5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="buttonCalc">
       <property name="text">
        <string>=</string>
       </property>
       <property name="operator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="button7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="exitButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(165, 29, 45);
	qproperty-icon: url(:/images/onoff.png);
	qproperty-iconSize: 32px 32px;
}

QPushButton:hover{
	background-color: rgb(237, 51, 59);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="buttonSubtract">
       <property name="text">
        <string>-</string>
       </property>
       <property name="operator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="button1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="button9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="button4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>+</string>
       </property>
       <property name="operator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="buttonDivide">
       <property name="text">
        <string>/</string>
       </property>
       <property name="operator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="clearButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(255, 120, 0);
}

QPushButton:hover {
	background-color: rgb(255, 163, 72);
}</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="buttonSqrt">
       <property name="text">
        <string>√</string>
       </property>
       <property name="operator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="buttonSquare">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font-style: italic;
}</string>
       </property>
       <property name="text">
        <string>x²</string>
       </property>
       <property name="operator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="powContainer" native="true">
       <widget class="QPushButton" name="buttonPower">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>54</width>
          <height>54</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="operator" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="powerLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>7</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	font: 700 14pt &quot;Ubuntu Sans&quot;;
	qproperty-alignment: 'AlignCenter';
	background: transparent;
	font-style: italic;
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span&gt;X&lt;/span&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lcdLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>40</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 2px solid rgb(36, 31, 49);
	background-color: rgb(181, 131, 90);
	color: rgb(0, 0, 0);
	font: 24pt;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 500 italic 14pt &quot;Z003&quot;;
}</string>
    </property>
    <property name="text">
     <string>Qasío</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
